syntax/parser.go

```diff
@@ -44,7 +44,7 @@ expr   -> C* amp="&"* @ C* arrow=(
         | C* "{" C* set=(elt=@:",",?) "}" C*
         | C* "{" C* dict=((key=@ ":" value=@):",",?) "}" C*
         | C* cond=("cond" "(" (key=@ ":" value=@):",",? ("*" ":" f=expr ","?)? ")") C*
-        | C* cond=(("(" control_var=expr ")" | IDENT)? C* "cond" "(" (key=@ ":" value=@):",",? ("*" ":" f=expr ","?)? ")") C*
+        | C* cond=(("(" control_var=expr ")" | IDENT)? C* "cond" "(" (key=(("(" @:"," ")") | @) ":" value=@):",",? ("*" ":" f=expr ","?)? ")") C*
```

Comment:
Tag this file as generated via git attributes. https://github.com/github/linguist#generated-code